linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_forwarding/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+prodtype: sle15
+
+title: 'Disable Kernel Parameter for IPv6 Forwarding by default'
+
+description: '{{{ describe_sysctl_option_value(sysctl="net.ipv6.conf.default.forwarding", value="0") }}}'
+
+rationale: |-
+    IP forwarding permits the kernel to forward packets from one network
+    interface to another. The ability to forward packets between two networks is
+    only appropriate for systems acting as routers.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85716-9
```

Comment:
This cce is already used in a different file. You can check this by running
```
./build_product sle15
cd build
ctest -j4 -L quick
```
This will run some quick tests, one of them is check for unique cces.
This is the output:
```
cce CCE-85716-9 is included in files: 
 - linux_os/guide/system/auditing/auditd_configure_rules/audit_execution_selinux_commands/audit_rules_execution_chcon/rule.yml
 - linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_forwarding/rule.yml
 - ```

Reply:
Fixed.